provisioning/templates/nginx-vhost.conf.j2

```diff
@@ -34,7 +34,7 @@ server {
         fastcgi_intercept_errors on;
         fastcgi_read_timeout {{ php_max_execution_time }};
         include fastcgi_params;
-        fastcgi_pass {{ php_fpm_listen }};
+        fastcgi_pass unix:{{ php_fpm_listen }};
```

Comment:
We could also just stick with tcp sockets. Or are the performance benefits of unix sockets noticeable?